### PR TITLE
Allow empty name for actions

### LINKED ISSUE #269: [Bug] nheko actions didnt get pick up
**Describe the bug**
in the instant messaging app nheko, the actions of notifications that are from nheko never get picked up. that mean clicking on the notif doesn't direct to chat room, and inline reply feature doesn't get triggered. 

**To Reproduce**
1. Open nheko
2. Receive a message (no inline)
3. clicking on it
4. nheko window doesn't get spawned, nor the existing window direct to incoming message.

**Expected behavior**
- Clicking onto nheko messages spawn the window and jump to the conversation interface. 
- inline reply works with it.

**Logs**
```
** (swaync:1497): DEBUG: 23:18:34.349: notiDaemon.vala:171: Notification: 
category:		im.received
image_data:		true
expire_timeout:		-1
replaces_id:		0
desktop_entry:		nheko
body:			kyy : hi
app_icon:		
action_icons:		false
time:			1685373514
hints:		
	image-data: data
	sound-name: 'message-new-instant'
	desktop-entry: 'nheko'
	category: 'im.received'
applied_id:		1
image_path:		(null)
inline-reply:		(null)
app_name:		nheko
summary:		kyy 
icon_data:		false
resident:		false
default_action:		(null)
actions:		
urgency:		Low
```

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Version 0.9.0-1 with inline reply config on

**Additional context**
Tried with dunst and kde, both doesn't have the problem

### PR DESCRIPTION
**Pull request by slikie**
fix #269

with this inline reply and click to jump to dialog works perfectly

### CODE REVIEW

File: src/notiModel/notiModel.vala [hunk 1]
```diff
@@ -241,15 +241,16 @@ namespace SwayNotificationCenter {
...
-
+                    if (action.name != null && action.identifier != null ) {
                         string id = action.identifier.down ();
                         switch (id) {
                             case "default":
                                 default_action = action;
                                 break;
                             case "inline-reply":
+                                if (action.name == "") {
+                                    action.name = "reply";
```
Comment (ErikReider) [inline src/notiModel/notiModel.vala:252]: We should make the default capitalized, so "Reply"

File: src/notiModel/notiModel.vala [hunk 2]
```diff
@@ -241,15 +241,16 @@ namespace SwayNotificationCenter {
                     var action = new Action ();
                     action.identifier = actions[i];
                     action.name = actions[i + 1];
-                    if (action.name != null && action.identifier != null
-                        && action.name != "" && action.identifier != "") {
-
+                    if (action.name != null && action.identifier != null ) {
```
Comment (ErikReider) [inline src/notiModel/notiModel.vala:244]: Remove whitespace after `action.identifier != null`

**Review by ErikReider — CHANGES_REQUESTED** [review]
Just some minor nits

**Comment by slikie** [comment]
fixed

**Review by ErikReider — APPROVED** [review]
LGTM! Thanks :)

**Pull request merged**